### PR TITLE
Nbam 55 59 86

### PR DESCRIPTION
**Pull request by jaakulan**
Fixed Sidebar issues

### CODE REVIEW

File: WebApp/webapp/src/components/Layout/index.module.css [hunk 1]
```diff
@@ -2,11 +2,6 @@
     display: flex;
 }
 
-.navbar {
-    flex: 1 0 10vw;
-    background-color: var(--navbar-colour);
-}
```
Comment (parathan) [inline WebApp/webapp/src/components/Layout/index.module.css:8]: keeping this in prevents a white bar from appearing when resizing the application

File: WebApp/webapp/src/components/NavBar/index.module.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  .expandedOverlay {
+    transition: opacity 0.3s ease;
+  }
+  
+  .companyInfo {
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    gap: 1rem;
+    color: #dddddd;
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.module.css:14]: can you change this to the global variables used for the css colors ie --var(--header-color) etc. found in the app.css file.
Any colours used in css files should be written there so theme colours aren't broken

File: WebApp/webapp/src/components/NavBar/index.module.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  }
+  
+  .menuSubtitle {
+    padding: 1rem;
+    color: #dddddd;
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.module.css:19]: same as above

File: WebApp/webapp/src/components/NavBar/index.module.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  }
+  
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.module.css:28]: same as above

File: WebApp/webapp/src/components/NavBar/index.module.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    z-index: 1;
+  }
+  
+  .grayLine {
+    height: 1px;
+    background-color: #ccc;
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.module.css:34]: same as above

File: WebApp/webapp/src/components/NavBar/index.module.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    margin: 20px 0;
+  }
+  
+  .activeMenuItem {
+    background-color: #333333;
+    color: #111633;
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.module.css:40]: same as above:
also can you change the color to white, or whatever the standard navbar font is when its not clicked. Text is hard to see when navbar item is selected

File: WebApp/webapp/src/components/NavBar/index.tsx [hunk 7]
```diff
@@ -1,35 +1,91 @@
...
+          color: '#dddddd',
+          height: '100vh',
+          position: 'fixed',
+          width: '14%'
+        },
+      }}
+    >
+      <Box className={styles.companyInfo}>
+        <FaBuilding size={30} />
+        <Typography variant="h6">NBAnalytics</Typography>
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.tsx:48]: Change this back to NBAMatchups for now

File: WebApp/webapp/src/pages/Home/index.tsx [hunk 8]
```diff
@@ -23,7 +23,7 @@ function Home() {
       <div className={styles.homePage}>
         <header className={styles.header}>
             <p>
-              Welcome to NBAMatchups
+              Welcome to NBAnalytics
```
Comment (parathan) [inline WebApp/webapp/src/pages/Home/index.tsx:26]: Change this back to NBAMatchups for now

**Review by parathan — CHANGES_REQUESTED** [review]
LGTM
Just some quick fixes

File: WebApp/webapp/src/components/NavBar/index.tsx [hunk 9]
```diff
@@ -1,35 +1,91 @@
...
+      </Box>
+      <Box className={styles.menuSubtitle}>
+        <Typography variant="subtitle1">Menu</Typography>
+      </Box>
```
Comment (parathan) [inline WebApp/webapp/src/components/NavBar/index.tsx:52]: Can remove this as well

Comment (jaakulan) [reply WebApp/webapp/src/components/NavBar/index.tsx:52]: i think this makes it look nicer tho

File: WebApp/webapp/src/components/Layout/index.module.css (hunk 1 above)
Comment (jaakulan) [reply WebApp/webapp/src/components/Layout/index.module.css:8]: fixed that without this

**Review by jaakulan — COMMENTED** [review]
Added css changes

File: WebApp/webapp/src/components/NavBar/index.tsx (hunk 9 above)
Comment (parathan) [reply WebApp/webapp/src/components/NavBar/index.tsx:52]: You should either take this out or give it different styling to differentiate it from the other tabs. Maybe a different background colour

**Review by parathan — CHANGES_REQUESTED** [review]

**Review by jaakulan — COMMENTED** [review]
changed Menu color

**Review by parathan — APPROVED** [review]

**Pull request merged**